js/apps/admin-ui/public/resources/en/user-federation-help.json

```
@@ -27,6 +27,7 @@
   "ldapSynchronizationSettingsDescription": "This section contains options related to synchronization of users from LDAP to the Keycloak database.",
   "syncRegistrations": "Should newly created users be created within LDAP store? Priority effects which provider is chosen to sync the new user. This setting is effectively appplied only with WRITABLE edit mode.",
   "importUsersHelp": "If true, LDAP users will be imported into the Keycloak DB and synced by the configured sync policies.",
+  "validateOnAccessHelp": "If on, import validation of a LDAP federation provider will be performed for each user every access. The validation checks if the user is still present and enabled in LDAP with dedicated LDAP queries. If off, then changes made in LDAP are only visible locally after explicit or periodic synchronization.",
```

This description might lack a word. I'm also not sure if it checks for enabled on validation; I think it checks just it it is present in LDAP. And then there is some logic in `LDAPStorageProvider#proxy()` which would need to be captured in this text. See below for a suggestion, wich might still be incomplete.

```suggestion
  "validateOnAccessHelp": "If on, import validation of a LDAP federation provider will be performed for each user on every access. The validation checks if the user is still present in LDAP with dedicated LDAP queries, and depending on the configuration uses the data from LDAP instead of previously cached data. If off, then changes made in LDAP are only visible locally after explicit or periodic synchronization.",
```

---

Done.